testing/evaluate_tests.py

```diff
@@ -113,9 +121,11 @@ def test_schema(resp, test_value, test_name=""):
         errs.update(res)
 
     if errs:
-        logger.info("Test {0} - Schema test failed for {1} with response {2}".format(
-            test_name, test_value, errs))
-        logger.info("Test {0} - Full response is {1}".format(test_name, resp))
```

Comment:
deleted the line `logger.info("Test {0} - Full response is {1}".format(test_name, resp))` since resp is a big dictionary which would be logged